_data/simple-icons.json

```diff
@@ -5065,6 +5065,11 @@
             "hex": "8B89CC",
             "source": "https://protonmail.com/media-kit"
         },
+        {
+            "title": "Protools",
```

Comment:
Avid refer to the product as `Pro Tools` as opposed to `Protools` - any chance you could update the naming in both `_data/simple-icons.json` and `icons/protools.svg` before merge?

Reply:
Sure :+1: I remember doubting about this at the time of opening the pull...